src/app/core/components/Navbar/AccountPopover.tsx

```diff
@@ -64,7 +64,7 @@ export default function AccountPopover({
       </Item>
       <Link
         to="/preferences"
-        className="flex cursor-pointer items-center py-2 px-3 text-gray-80 hover:text-gray-80 hover:bg-gray-1 active:bg-gray-5 no-underline"
+        className="flex cursor-pointer items-center py-2 px-3 text-gray-80 no-underline hover:bg-gray-1 hover:text-gray-80 active:bg-gray-5"
```

Comment:
Is this from the prettier auto save rule? or is there any change of classes here? 🤔